### ISSUE TITLE
How to implement weight decay towards the pre-trained model?

### ISSUE THREAD

**Issue by sedol1339** [NVIDIA/NeMo#10738]
Hello, let me one question.

If using NeMo for supervised fune-tuning, how do I implement penalizing the distance between starting and current weights? This was shown to be effective in https://arxiv.org/abs/1706.03610
